src/solver/EulerSolver.cpp

```diff
@@ -38,6 +38,13 @@ Solver::EulerSolver::EulerSolver(FlowField &localFlowField,
 	                                       localMesh,
 	                                       localMetrics,
 	                                       config.getMeshRefPoint());
+
+	// define time integration method
+	if (_config.getTemporalScheme() == Parser::SimConfig::TemporalScheme::RK5) {
+		_timeIntegrator = &Solver::EulerSolver::RungeKutta;
+	} else {
+		_timeIntegrator = &Solver::EulerSolver::EulerExplicit;
+	}
```

Comment:
This removes an if condition from the solver loop. Sounds good?

Reply:
Looks good to me